pkg/virt-handler/vm.go

```diff
@@ -586,10 +592,19 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 						if err != nil {
 							return err
 						}
-						if iface.PodIP != existingInterfaceStatusByName[domainInterface.Alias.Name].IP {
+
+						shouldCopy := !stringSlicesEqual(existingInterfaceStatusByName[domainInterface.Alias.Name].IPs, iface.PodIPs[:])
```

Comment:
Please consider using `reflect.DeepEqual`

Reply:
i did , but according this
https://yourbasic.org/golang/compare-slices/
the performance isnt good, but i can try to consider again (since its just 2 elements, in case we fine with the little performance impact it might have), first stabilizing the other cosmetics.

Reply:
We are already using `reflect.DeepEqual` in the "core" code. Even this file already has 3 usages of it.